app/forms/journeys/teacher_student_loan_reimbursement/still_teaching_form.rb

```diff
@@ -0,0 +1,40 @@
+module Journeys
+  module TeacherStudentLoanReimbursement
+    class StillTeachingForm < Form
+      attribute :employment_status
+      attribute :current_school_id
+
+      validates :employment_status,
+        presence: {
+          message: ->(form, _) { form.i18n_errors_path("select_which_school_currently", school_name: form.claim_school_name) }
```

Comment:
As per your existing comment, this error message probably should also revert to the TPS school in that scenario.